src/components/WarehouseInfo/WarehouseInfo.jsx

```diff
@@ -0,0 +1,68 @@
+import "./WarehouseInfo.scss";
+import arrowBack from "/assets/icons/arrow_back-24px.svg";
+import editIcon from "/assets/icons/edit-white-24px.svg";
+import { Link } from "react-router-dom";
+
+export default function WarehouseInfo() {
+	const data = {
+		id: 2,
+		warehouse_name: "Washington",
+		address: "33 Pearl Street SW",
+		city: "Washington",
+		country: "USA",
+		contact_name: "Greame Lyon",
+		contact_position: "Warehouse Manager",
+		contact_phone: "+1 (646) 123-1234",
+		contact_email: "glyon@instock.com",
+	};
+
+	return (
+		<div className="warehouse">
+			<section className="warehouse__title-card">
+				<div className="warehouse__title-wrapper">
+					<Link to={`warehouses`} className="warehouse__link">
```

Comment:
Clicking the back button currently routes to:
http://localhost:5173/warehouses/1/warehouses

It should route to
http://localhost:5173/warehouses

Reply:
Resolved